protected/humhub/modules/user/models/forms/EditGroupForm.php

```diff
@@ -55,18 +55,37 @@ public function afterFind()
         }
     }
 
+    /**
+     * @inheritdoc
+     */
     public function rules()
     {
         $rules = parent::rules();
         $rules[] = [['defaultSpaceGuid'], 'safe'];
         $rules[] = [['updateSpaceMemberships'], 'boolean'];
-        $rules[] = [['name', 'type'], 'required', 'except' => self::SCENARIO_MANAGER];
-        $rules[] = [['managerGuids', 'show_at_registration', 'show_at_directory'], 'safe', 'except' => self::SCENARIO_MANAGER];
-        $rules[] = [['subgroups', 'parent'], 'safe', 'except' => self::SCENARIO_MANAGER];
-        $rules[] = [['type'], 'validateTypeGroups', 'except' => self::SCENARIO_MANAGER];
+        $rules[] = [['name', 'type'], 'required'];
+        $rules[] = [['managerGuids', 'show_at_registration', 'show_at_directory'], 'safe'];
+        $rules[] = [['subgroups', 'parent'], 'safe'];
+        $rules[] = [['type'], 'validateTypeGroups'];
         return $rules;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function scenarios()
+    {
+        $scenarios = parent::scenarios();
+
+        $scenarios[self::SCENARIO_MANAGER][] = 'updateSpaceMemberships';
+
+        if ($this->isNewRecord) {
```

Comment:
Not tested, but couldn't we use something like
'$rules[] = [['updateSpaceMemberships'], 'boolean', 'when' => ($this->isNewRecord)];'
instead of moving some rules into the `scenario` section?

Reply:
@luke- I tried to use the option/filter `when` but it doesn't do any effect to a result of the methods `isAttributeActive()` and `activeAttributes()` because the result is built from the method `scenarios()` which use only rule options/filters `on` and `except`.